### DISCUSSION TITLE
hogwarts legecy

### DISCUSSION THREAD

**Discussion by toeegg** [UE4SS-RE/RE-UE4SS#604] [General]
Im posting this here becuase I dont understand the issue templete. none of my ue4ss mods for hogwarts legecy will work when I look at the consoles they say that I need to override the engeine version could someone please help me create a proper issue report for the matter.
[gui console.txt](https://github.com/user-attachments/files/16416366/gui.console.txt)
[[162941] Console created.txt](https://github.com/user-attachments/files/16416372/162941.Console.created.txt)

**Comment by UE4SS** [comment]
Search before asking for help.
See #601 and #597
TLDR: You need to use the experimental version (a link exists in one of the linked issues) and then you need to manually set the engine version in UE4SS-settings.ini as mentioned in one of the issues.

**Reply by toeegg** [reply]
great thanks
